frontend/components/sections/BitTableSection.tsx

```diff
@@ -111,55 +100,140 @@ function Bit({ type, driverSizeColor, sizes }: GroupedBit) {
                width={20}
                height={20}
             />
-            <Flex
-               px="1"
-               h="4"
-               bg={`${driverSizeColor}.100`}
-               borderRadius="full"
-               borderColor={`${driverSizeColor}.200`}
-               borderWidth={1}
-               alignItems="center"
-               justify="center"
-               minW="10"
-            >
-               <Text
-                  as="span"
-                  color={`${driverSizeColor}.700`}
-                  fontSize="xs"
-                  fontFamily="mono"
-               >
-                  {type.driverSize}
-               </Text>
-            </Flex>
-         </Flex>
-         <Flex flexGrow={1} alignItems="center" gap="2.5" m="2.5">
-            <Text flexGrow={1}>{type.name}</Text>
-            <Flex flexWrap="wrap" gap={1} justifyContent="flex-end">
-               {sizes.map((size) => (
+            <Flex flexDirection="column" gap="1">
+               {Object.keys(sizes).map((driverSize) => (
                   <Flex
-                     key={size}
-                     px="1.5"
+                     key={driverSize}
+                     px="1"
                      h="4"
-                     bg="brand.100"
+                     bg={`${driverSizeColors[driverSize]}.100`}
                      borderRadius="full"
-                     borderColor="brand.200"
+                     borderColor={`${driverSizeColors[driverSize]}.200`}
                      borderWidth={1}
                      alignItems="center"
                      justify="center"
-                     minW="9"
+                     minW="10"
                   >
                      <Text
                         as="span"
-                        color="brand.700"
+                        color={`${driverSizeColors[driverSize]}.700`}
                         fontSize="xs"
                         fontFamily="mono"
                      >
-                        {size}
+                        {driverSize}
                      </Text>
                   </Flex>
                ))}
             </Flex>
          </Flex>
+         <Flex flexGrow={1} alignItems="center" gap="3" p="3">
+            <Flex gap="1">
+               {totalBitCount > 0 && (
```

Comment:
Would
```suggestion
               {totalBitCount && (
```
be reasonable?

Reply:
I tend to prefer explicit numeric comparison if the variable is numeric.
My reasoning is that, if the data type is changed in the upper layers from a numeric 0 to a `"0"` string, the outcome of that check would be different.
The `>` operator would raise an error while the simple variable check would not.

Reply:
I don't have a strong opinion. I think I'd prefer it changed just slightly, and we can type `totalBitCount` as numeric if you're concerned.